dockcross

```diff
@@ -0,0 +1,234 @@
+#!/bin/bash
+
+DEFAULT_DOCKCROSS_IMAGE=dockcross/linux-armv5
+
+#------------------------------------------------------------------------------
+# Helpers
+#
+err() {
+    echo -e >&2 ERROR: $@\\n
+}
+
+die() {
+    err $@
+    exit 1
+}
+
+has() {
+    # eg. has command update
+    local kind=$1
+    local name=$2
+
+    type -t $kind:$name | grep -q function
+}
+
+#------------------------------------------------------------------------------
+# Command handlers
+#
+command:update-image() {
+    docker pull $FINAL_IMAGE
+}
+
+help:update-image() {
+    echo Pull the latest $FINAL_IMAGE .
+}
+
+command:update-script() {
+    if cmp -s <( docker run $FINAL_IMAGE ) $0; then
```

Comment:
Don't think so... it's looking wether the script `dockcross` is up to date, isn't it?